views/components/ConfigurationCard/index.jsx

```diff
@@ -73,7 +74,9 @@ const ConfigurationCard = ({
           {ownsConfig ? (
             <>
               <IconButton
-                onClick={() => openNewWindow(routes.editConfiguration(_id))}
+                onClick={() =>
+                  onEditOrViewConfig(routes.editConfiguration(_id))
```

Comment:
this is just wrapping `navigate` in another prop. The `ConfigurationCard` shouldn't need to know about the routes, it can just call `onEditConfig(config)`. Same for the "view" piece, it can call `onClickConfig(config)` or `onViewConfig(config)`.

Reply:
Oh sorry, it's confusing I used gridly as the example for this and it passed the navigate propped. Thanks for clearing that up

Reply:
[5d91666](https://github.com/AMP-SCZ/dpdash/pull/317/commits/5d9166607f908fb96fdf762cb55dc41547e7ee00)